MetaMask/metamask-extension
# Find and replace of all typography colors to design tokens

Fixes: #13774

Explanation: Updates all instances of Typography with the correct design token options. This should be a simple find and replace. I have another PR that contains all of the component updates that these files touch but thought this might be easier to review first.

Manual testing steps:
  - Go to the [Typography storybook docs](https://metamask.github.io/metamask-storybook/index.html?path=/docs/ui-components-ui-typography-typography-stories-js--color)
  - See all the valid colors for the Typography component or check [this list](https://github.com/MetaMask/metamask-extension/blob/develop/ui%2Fcomponents%2Fui%2Ftypography%2Ftypography.js#L17-L31)
  - In the code base do a search for `color={COLORS.` make sure you hit the `Aa` icon otherwise it will include `backgroundColors={`
  - Make sure I haven't missed any 😅 
  
```
COLORS.BLACK => COLORS.TEXT_DEFAULT
COLORS.UI4, COLORS.NEUTRAL_GREY => COLORS.TEXT_ALTERNATIVE
COLORS.UI1, COLORS.UI2, COLORS.UI3 => COLORS.TEXT_MUTED
COLORS.PRIMARY1 => COLORS.PRIMARY_DEFAULT
COLORS.SUCCESS3 => COLORS.SUCCESS_DEFAULT
COLORS.ERROR1 => COLORS.ERROR_DEFAULT
```
